src/core/DropdownMenu/style.ts

```diff
@@ -0,0 +1,196 @@
+import styled from "@emotion/styled";
+import { Autocomplete } from "@material-ui/lab";
+import InputSearch from "../InputSearch";
+import MenuItem from "../MenuItem";
+import {
+  CommonThemeProps,
+  fontBodyXs,
+  fontBodyXxs,
+  fontCapsXxxxs,
+  getBorders,
+  getColors,
+  getSpaces,
+} from "../styles";
+
+export const StyledMenuItem = styled(MenuItem)`
+  width: 100%;
+  padding: 0;
+  min-height: unset;
+  white-space: pre-wrap;
+
+  ${(props: StyleProps) => {
+    const { count } = props;
+
+    if (count) {
+      return `
+        > span > span {
```

Comment:
relying on HTML structure for styling is brittle. Is there a way we can target a class name instead? Thank you!

Reply:
Oops :D
Replaced with a proper class name 💯